facebook/facebook.c

```diff
@@ -26,6 +26,9 @@
 #define OPT_SELFMESSAGE 0
 #endif
 
+#define FB_NEW_BUDDY  1
```

Comment:
Let's prefix these with `FB_BITPTR_` to avoid conflicts in other namespaces. Also, an `enum` might be better suited here.